.github/workflows/test.yml

```diff
@@ -25,6 +25,13 @@ jobs:
       - name: Serialize seed
         run: python seed/serialize.py seed/seed.json
 
+      - name: Generate testing_fieldtrials.json for perf tests
+        run: python3 seed/fieldtrials_testing_config_generator.py
+          --output fieldtrial_testing_config.json
+          --output-revision brave-variations-revision.txt
+          --target-date "2024-08-12 00:02:25 +0000"
```

Comment:
this is not enough. the test should also run on the current revision. pls update the script to do this, i.e. instead of `--target-*` flags there can be `--commit` flag.